requirements.txt

```diff
@@ -1,5 +1,5 @@
 cryptography==37.0.4
-ops==1.5.0
+ops==1.5.4
```

Comment:
Possibly what is causing the new line in the CSRs?

Reply:
time for ops==2.0.0rc2 ? ;-)

Reply:
We should retry [PR#54](https://github.com/canonical/postgresql-operator/pull/54) to see if that was the issue there as well.

Reply:
ACK, step-by-step. LGTM here.